README.md

```diff
@@ -1,5 +1,31 @@
 # RayFed
-A multiple parties joint, distributed execution engine on the top of Ray.
+A multiple parties joint, distributed execution engine based on Ray, to help build your own federated learning frameworks in minutes.
+
+
+## Features
+- **Ray Native Programming Pattern**  
+  
+  Let your write your federated and distributed computing applications like writing a single machine program.
+
+- **Multiple Controller Execution Mode**  
+  
+  The RayFed job can be ran in single controller mode when developing and debugging, while it will be ran in multiple controller mode in production without any code change.
```

Comment:
"be ran" -> "be run"

Reply:
Good catch

Reply:
The RayFed job can be run in the single-controller mode for developing and debugging and the multiple-controller mode for production without code change